velox/common/dynamic_registry/DynamicLibraryLoader.h

```diff
@@ -38,6 +38,10 @@ namespace facebook::velox {
 ///
 /// Loading a library twice can cause a components to be registered twice.
 /// This can fail for certain Velox components.
-void loadDynamicLibrary(const std::string& fileName);
+
+static constexpr const char* kSymbolName = "registry";
```

Comment:
I doubt we need a constant for function default values. We should be able to inline these.